src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2948,7 +2957,7 @@ public void Free()
 
         protected sealed class DeclaredMembersAndInitializers
         {
-            public readonly ImmutableArray<Symbol> NonTypeMembers;
+            public readonly ImmutableArray<Symbol> NonTypeMembersWithPartialImplementations;
```

Comment:
It might be good to also rename the corresponding field of `DeclaredMembersAndInitializersBuilder`.